NavigationReactNative/src/NavigationBar.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {ReactElement} from 'react';
+import {requireNativeComponent, Image, Platform, UIManager} from 'react-native';
+import LeftBar from './LeftBar';
+import RightBar from './RightBar';
+
+var NVNavigationBar = requireNativeComponent<any>('NVNavigationBar', null);
+var NavigationBar = ({hidden, logo, navigationImage, overflowImage, children, style, ...otherProps}) => {
+    if (Platform.OS === 'android' && hidden)
+        return null
+    var menuItems = (React.Children.toArray(children) as ReactElement<any>[])
+        .filter(({type}) => (type === LeftBar || type === RightBar))
+        .sort((a, b) => (a.type === b.type ? 0 : (a.type === RightBar ? 1 : -1)))
+        .map(({props}) => (
+            (React.Children.toArray(props.children) as ReactElement<any>[])
+                .map(({props}) => ({
+                    ...props,
+                    show: Platform.OS === 'android'
+                     ? UIManager.getViewManagerConfig('NVNavigationBar').Constants.ShowAsAction[props.show]
+                     : undefined,
+                    image: Image.resolveAssetSource(props.image),
+                })
+            )
+        ))
+        .reduce((a, b) => a.concat(b), [])
+    return (
+        <NVNavigationBar
+            menuItems={menuItems}
+            logo={Image.resolveAssetSource(logo)}
+            navigationImage={Image.resolveAssetSource(navigationImage)}
+            overflowImage={Image.resolveAssetSource(overflowImage)}
+            style={[Platform.OS === 'android' ? { height: 50 } : undefined, style]}
```

Comment:
Change to the style prop as follows
```jsx
style={Platform.OS === 'android' ? {height: 56} : undefined}
```
There are two changes here
1. The height is 56 to match the [material design specs](https://material.io/components/app-bars-top/#specs)
2. Removed the user style prop. If we let the user customise the style then it we could accidentally break their code when we add new features, e.g., collapsing toolbar.

Reply:
If the user provides a style prop then at the moment we're still rendering it because we include the ```{...otherProps}```. We can either:
1. Move {...otherProps} before the our style prop so that the user style is ignored
2. Destructure the user style prop and ingore it